app_feup/lib/controller/networking/network_router.dart

```diff
@@ -231,6 +222,21 @@ class NetworkRouter {
     return tripList;
   }
 
+  static String getBusTimeRemaining(rawBusInformation){
+    final busTimeOfArrival = rawBusInformation[1].text.trim();
+    var busTimeRemaining = '0';
+
+    if(busTimeOfArrival != 'a passar'){
```

Comment:
Since we are now in a separated function, we can use the if to return the 0 right off the bat or make the more complicated calculation, thus removing the need for a var and improving readability. Also, we can make this function return an int already. Wdyt?

Reply:
I was hesitant about returning an int since in the future we might need to stop parsing the estimated time remaining from string to int because this operation was only used to calculate the estimated time of arrival (= time now + estimated time remaining); this procedure becomes useless since this value is already provided by this API (but was not by the previous one). In this case it would be easier to change a simple parseInt than to change a functions return. 
But since that is a motivation for another issue, I agree that, for readability, that is better, I will change it for the next commit, as well as returning 0 right off the bat.